rfcs/2021-12-20-10517-llvm-backend-for-vrl.md

```diff
@@ -649,6 +649,34 @@ to
 We use the same strategy that is used in our Vector binary, which currently uses
 the default "unwind".
 
+### Testing Strategy
+
+Unit tests: Add tests for the code generation of each expression in isolation,
+making sure that the emitted LLVM IR passes static analysis and the generated
+code produces the expected result. This covers edge cases specific to each
+expression.
+
+Behavior tests: Make sure that the existing test corpus living in
+[`tests/behavior/transforms/remap.toml`](https://github.com/vectordotdev/vector/blob/master/tests/behavior/transforms/remap.toml)
+passes when using the LLVM based execution engine.
```

Comment:
I think the more relevant tests live in `lib/vrl/tests`. The ones in `remap.toml` are mainly meant to validate the interface between Vector and VRL.

Reply:
Adressed in 9314f8e81ca5e7ba70f8633e3b897d54dd992d8c.